packages/foundry/src/cloneProjectTemplate.ts

```diff
@@ -0,0 +1,8 @@
+import degit from "degit";
+import { ensureDir } from "fs-extra";
+
+export async function cloneProjectTemplate (outputDirectory: string, repositoryUrl: string) {
+    await ensureDir(outputDirectory);
```

Comment:
Also wonder if you really need to do this?

My guess is that the `clone` function called right after this will either ensure the directory is OK or throw an error at execution if the directory is not OK.

Also, how do you communicate to the client that the generation failed if the directory is not OK?

Reply:
Honestly i think we should. I don't see a way to properly forward error other than that